src/asset-swapper/utils/market_operation_utils/path.ts

```diff
@@ -59,6 +59,10 @@ export class Path {
         protected readonly adjustedSize: PathSize,
     ) {}
 
+    public hasTwoHop(): boolean {
+        return (this.sourceFlags & SOURCE_FLAGS[ERC20BridgeSource.MultiHop]) > 0;
```

Comment:
is this ok to be different than the previous logic `path.sourceFlags === SOURCE_FLAGS[ERC20BridgeSource.MultiHop]`?

Reply:
yes it's intentional! (I don't like the flag-based approach and want to change it to a set in the future).